src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -349,7 +435,31 @@ def list_wl_recovery_points(cmd, client, resource_group_name, vault_name, item,
     recovery_points = client.list(vault_name, resource_group_name, fabric_name, container_uri, item_uri, filter_string)
     paged_recovery_points = cust_help.get_list_from_paged_response(recovery_points)
 
-    return paged_recovery_points
+    fetch_tier(paged_recovery_points)
+
+    recovery_point_list = check_rp_move_readiness(paged_recovery_points, target_tier, is_ready_for_move)
+    recovery_point_list = filter_rp_based_on_tier(recovery_point_list, tier)
+    return recovery_point_list
+
+
+def move_wl_recovery_points(cmd, resource_group_name, vault_name, item_name, rp_id, source_tier,
+                            destination_tier):
+
+    container_uri = cust_help.get_protection_container_uri_from_id(item_name.id)
+    item_uri = cust_help.get_protected_item_uri_from_id(item_name.id)
+
+    Dict = dict({'VaultStandard': 'HardenedRP', 'VaultArchive': 'ArchivedRP', 'Snapshot': 'InstantRP'})
```

Comment:
please don't do generic naming. give a name that is specific to this case and easily understandable.

Reply:
done